index.html

```diff
@@ -0,0 +1,156 @@
+<!doctype html>
+<head>
+  <title>Static Web HTML and CSS Quiz</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+
+<body>
+
+
+<!-- 1st CARD --> 
+
+  <div class="card">
```

Comment:
Requirement 1 asks that "Each card should be an `<article>`" instead of a `<div>`